examples/templateservicebroker/apiserver-config.yaml

```diff
@@ -0,0 +1,4 @@
+kind: TemplateServiceBrokerConfig
+apiVersion: config.templateservicebroker.openshift.io/v1
+templateNamespaces:
+- openshift
```

Comment:
Keeping it origin makes it more likely that it's maintained by the developer messing with the config. Like this, it's possible to write a linter keeping it in sync with comments.